packages/ui/src/views/canvas/LightTooltip.js

```diff
@@ -0,0 +1,12 @@
+import { styled } from '@mui/material/styles'
```

Comment:
Do u think it make sense to refractor this into `ui-component/tooltip` and maybe rename as smtg that is specific like `NodeTooltip` as its only used for `CanvasNode` right?

Reply:
I've moved the component to `ui-component/tooltip` and renamed it to `NodeTooltip`.